tests/schemas/test_invalid_survey_id_whitespace.json

```diff
@@ -0,0 +1,3364 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "lms ",
+    "title": "Labour Market Survey (Alpha)",
```

Comment:
we don't need such a big test_schema for this test, again i suppose it makes no difference